src/generators/dot_net/index.js

```diff
@@ -236,8 +238,8 @@ class DotNet extends Generator {
       }
     } else {
       // In the model everything is one or the other (at a minimum must inherit https://schema.org/Thing)
-      // throw new Error("No base class specified for: " + model.type);
-      return "None";
+      throw new Error("No base class specified for: " + model.type);
+      // return "None";
```

Comment:
I think just blow away this commented out line as without any context it's only going to confuse people in the future.